.github/workflows/build-ios.yml

```diff
@@ -56,7 +56,7 @@ jobs:
             example/ios/Pods
             ~/Library/Caches/CocoaPods
             ~/.cocoapods
-          key: build-ios-pods-${{ matrix.react-native-architecture }}-${{ hashFiles('example/node_modules/react-native/package.json') }}
```

Comment:
Explanation in https://github.com/Expensify/react-native-live-markdown/pull/529/files#r1819203272